psi4/driver/plugin.py

```diff
@@ -87,7 +88,7 @@ def sanitize_name(name):
 
 # Determine the available plugins
 available_plugins = []
-psidatadir = os.environ.get('PSIDATADIR', None)
+psidatadir = core.get_datadir()
```

Comment:
We keep building up functions off `core`. In the future it could be nice to move this off to something else like `core.environment.get_datadir()`.

Reply:
Yeah, I'm hoping this particular fn will be short-lived.